src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -411,7 +413,7 @@ void AutomatedRssDownloader::on_exportBtn_clicked()
         return;
     }
     // Ask for a save path
-    QString save_path = QFileDialog::getSaveFileName(this, tr("Where would you like to save the list?"), QDir::homePath(), tr("Rules list (*.rssrules)"));
+    QString save_path = QFileDialog::getSaveFileName(this, tr("Export RSS rules to file"), QDir::homePath(), tr("qBittorrent (*.rssrules)"));
```

Comment:
"qBittorrent" what? "qBittorrent rules"? Same question to the constants in  `AutomatedRssDownloader::on_importBtn_clicked()`